test/testvm.py

```diff
@@ -158,7 +188,8 @@ def build(self, args):
 
     def run_setup_script(self, script, args):
         """Prepare a test image further by running some commands in it."""
-        self.start(maintain=True)
+        # run this on the original image, not the one in the run directory
+        self.start(maintain=True, original=True)
```

Comment:
you're right - I didn't consider our hubbot setup. In my testing environment, I don't clean up ./run before I create a new image.